src/components/Card/Card.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  disabled?: boolean;
+  error?: boolean;
+  imgSrc: string;
+  imgAlt: string;
+  tagColor?: string;
+  tagBg?: string;
```

Comment:
```suggestion
  img: {
    src: string;
    alt: string;
  };
  tag?: {
    color: string;
    background?: string;
  };
```
Zamiast takiej płaskiej struktury warto sobie tworzyć obiekty, które są bardziej reużywalne. Jeśli `img` czy `tag` będzie używany też w innych komponentach to łatwo sobie później zrobić z tego Interface.